Makefile

```diff
@@ -6,7 +6,7 @@ audius: main.go
 
 build-docker:
 	@echo "Building Docker image..."
-	docker buildx build --load --build-arg NETWORK=stage -t audius/dot-slash:dev .
+	docker buildx build --load --build-arg NETWORK=stage --build-arg BRANCH=as/dot-slash-audius -t audius/dot-slash:dev .
```

Comment:
needed to make an audius-docker-compose change for this work, details in this PR 
https://github.com/AudiusProject/audius-docker-compose/pull/387